demo/index.html

```diff
@@ -30,87 +31,103 @@
-      transition: all 0.2s;
-      -webkit-transition: all 0.2s;
-    }
-
-    iron-icon:hover {
-      fill: var(--google-yellow-700);
-    }
-
-    .set {
-      margin: auto;
-      padding: 1em 0;
-      border-bottom: 1px solid silver;
-      @apply(--layout-horizontal);
-      @apply(--layout-wrap);
-    }
-
-    .set:last-of-type {
-      border-bottom: none;
-    }
-
-    .set:nth-of-type(4n-3) {
-      color: var(--paper-grey-700);
-    }
-
-    .set:nth-of-type(4n-2) {
-      color: var(--paper-pink-500);
-    }
-
-    .set:nth-of-type(4n-1) {
-      color: var(--google-green-500);
-    }
-
-    .set:nth-of-type(4n) {
-      color: var( --google-blue-500);
-    }
-
-    .container {
-      min-width: 10em;
-      padding: 1em 0.5em;
-      text-align: center;
-      @apply(--layout-vertical);
-      @apply(--layout-center);
-      @apply(--layout-flex);
-    }
-
-    .container > div {
-      margin-top: 0.5em;
-      color: black;
-      font-size: 10px;
-    }
-  </style>
 </head>
 <body>
-  <template is="dom-bind">
-   <iron-meta type="iconset" list="{{iconsets}}"></iron-meta>
-
-    <template is="dom-repeat" items="{{iconsets}}">
-      <h2>{{item.name}}</h2>
-
-      <div class="set">
-        <template is="dom-repeat" items="{{getIconNames(item)}}">
-          <span class="container">
-            <iron-icon icon="{{item}}"></iron-icon>
-            <div>{{item}}</div>
-          </span>
-        </template>
-      </div>
-    </template>
-  </template>
+  <custom-style>
+    <style is="custom-style">
+      h2 {
+        text-transform: capitalize;
+      }
+
+      iron-icon {
+        transition: all 0.2s;
+        -webkit-transition: all 0.2s;
+      }
+
+      iron-icon:hover {
+        fill: var(--google-yellow-700);
+      }
+
+      .set {
+        margin: auto;
+        padding: 1em 0;
+        border-bottom: 1px solid silver;
+        @apply --layout-horizontal;
+        @apply --layout-wrap;
+        @apply --layout-center-justified;
+      }
+
+      .set:last-of-type {
+        border-bottom: none;
+      }
+
+      .set:nth-of-type(4n-3) {
+        color: var(--paper-grey-700);
+      }
+
+      .set:nth-of-type(4n-2) {
+        color: var(--paper-pink-500);
+      }
+
+      .set:nth-of-type(4n-1) {
+        color: var(--google-green-500);
+      }
+
+      .set:nth-of-type(4n) {
+        color: var( --google-blue-500);
+      }
+
+      .container {
+        width: 10em;
+        margin: 1em 0.5em;
+        text-align: center;
+        @apply --layout-flex;
+        @apply --layout-vertical;
+        @apply --layout-center;
+        flex-basis: auto;
+      }
+
+      .container > div {
+        margin-top: 0.5em;
+        color: black;
+        font-size: 10px;
+      }
+    </style>
+  </custom-style>
 
   <script>
-    document.querySelector('[is=dom-bind]').getIconNames = function(iconset) {
-      return iconset.getIconNames();
-    };
+    window.addEventListener('WebComponentsReady', function() {
```

Comment:
```html
<iron-meta type="iconset" list="{{iconsets}}"></iron-meta>
```
.. doesn't work anymore because `list` isn't a property. Also, having the extra `<dom-repeat>` around the repeated items kind of screws up styling them since they're meant to flex and `display: contents;` doesn't actually work yet. The easiest way to handle this seemed to be to generate the DOM manually but I'm not attached to that method if you've got other recommendations. I tried changing it so that the `<dom-repeat>` elements were used for display instead of wrapping with divs but this seems to trigger a ShadyCSS problem that prevents all the styles on the page from being rewritten properly. I'll keep trying to revert this but the ShadyCSS thing is blocking it for now.